libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -40,26 +40,25 @@
 
 extern const AP_HAL::HAL& hal;
 
-
 //==============================================================================
 // Constants and Definitions
 //==============================================================================
 
-#define AP_FOLLOW_TIMEOUT_MS    3000    // position estimate timeout
 #define AP_FOLLOW_SYSID_TIMEOUT_MS 10000 // forget sysid we are following if we have not heard from them in 10 seconds
 
 #define AP_FOLLOW_OFFSET_TYPE_NED       0   // offsets are in north-east-down frame
 #define AP_FOLLOW_OFFSET_TYPE_RELATIVE  1   // offsets are relative to lead vehicle's heading
 
-#define AP_FOLLOW_ALTITUDE_TYPE_RELATIVE  1 // relative altitude is used by default   
+#define AP_FOLLOW_ALT_TYPE_DEFAULT      static_cast<float>(Location::AltFrame::ABOVE_HOME)
 
 #define AP_FOLLOW_POS_P_DEFAULT 0.1f    // position error gain default
 
+
 #if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
- #define AP_FOLLOW_ALT_TYPE_DEFAULT 0
- #define AP_FOLLOW_DIST_MAX_DEFAULT 0
+ #define AP_FOLLOW_TIMEOUT          0.6    // position estimate timeout (for Plane)
+ #define AP_FOLLOW_DIST_MAX_DEFAULT 500
```

Comment:
I'm concerned this may cause ship-landing to misbehave at long distances, I prefer this to stay at zero meaning unlimited (please check!)